test/BingoGame.Indexer.CA.Tests/Processors/PlayedProcessorTests.cs

```diff
@@ -160,11 +172,17 @@ public PlayedLogEventProcessorTests()
             },
             BlockTime = DateTime.UtcNow
         };
-        var bingoedLogEventProcessor = GetRequiredService<PlayedProcessor>();
-        
-        Assert.ThrowsAsync<PlayEventAlreadyHandledException>(async () =>
-        {
-            await bingoedLogEventProcessor.HandleEventAsync(logEventInfo, logEventContext);
-        }); 
+        var mockLogger = new Mock<ILogger<PlayedProcessor>>();
+        var mockRepository = new Mock<IAElfIndexerClientEntityRepository<BingoGameIndexEntry, TransactionInfo>>();
+        var mockContractInfoOptions = new Mock<IOptionsSnapshot<ContractInfoOptions>>();
+        var mockObjectMapper = new Mock<IObjectMapper>();
+        var playedProcessor = new PlayedProcessor(
+            mockLogger.Object,
+            mockRepository.Object,
+            mockContractInfoOptions.Object,
+            mockObjectMapper.Object
+        );
```

Comment:
What's this chunk doing? `playedProcessor` is never used.

Reply:
deleted, please check again